backend/app/models/settings_types.py

```diff
@@ -0,0 +1,13 @@
+from enum import Enum
+
+
+class AntiRecommenderType(str, Enum):
+    """An enum of anti-recommender types."""
+
+    open_ai = "OpenAI"
```

Comment:
Usual convention is to capitalize the enumerator `OPEN_AI`.

Reply:
All enumerators are now capitalized.